screenpy_selenium/target.py

```diff
@@ -26,10 +26,28 @@ class Target:
         Target.the("header search bar").located_by("div.searchbar")
 
         Target.the("welcome message").located_by('//h2[@name = "welcome"]')
+
+        Target().located_by((By.ID, "username-field"))
     """
 
-    target_name: str
-    locator: Union[None, Tuple[str, str]]
+    _description: Optional[str] = None
+    locator: Optional[Tuple[str, str]] = None
+
+    @property
+    def target_name(self):
+        if self._description is not None:
+            return self._description
+        if self.locator:
+            return self.locator[1]
+        return self._description
```

Comment:
I'm wondering if we should just say `return None` here to prevent some confusion. `self._description` here **is** `None`, i think reading this people would have to do a double-take to realize that's what's happening.

If someone subclassing `Target` needs to overwrite this property to make their default work, they'll have to overwrite the whole thing anyway. Returning `self._description` here doesn't do them any favors.

Reply:
Updated